lib/Installation/Partitioner/LibstorageNG/EditPartitionPage.pm

```diff
@@ -0,0 +1,51 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: The class introduces all accessing methods for Edit Partition
+# Page of Expert Partitioner Wizard, that are common for all the versions of the
+# page (e.g. for both Libstorage and Libstorage-NG).
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::EditPartitionPage;
```

Comment:
This definitely belongs to `lib/Installation/Partitioner/FormattingOptionsPage.pm`, in case some shortcuts are different, we should at least inherit from it and override only the parts which are different.